bitnami/minio/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: minio
 sources:
   - https://github.com/bitnami/bitnami-docker-minio
   - https://min.io
-version: 11.7.7
+version: 11.7.8
```

Comment:
Could you bump the minor version of the chart? It is adding a non-breaking feature